Add PyCon India 2021 announcement blog (#352)
pythonindia/inpycon-blog: content/blog/2021/announcing-pycon-india.md

Title: PyCon India is back online for 2021
Date: 2021-02-25 17:00
Author: Nikhil Maan (@Sc0rpi0n101)
Category: 2021
Slug: announce-pycon-india-2021
Summary: Announcing the return of PyCon India as an online conference for 2021.

> We hope you've had a safe and healthy year. 

Author: ...as 2020 was...?

> We hope you've had a safe and healthy year. 

Author: ```suggestion
From all of us at the PyCon India team, we wish you've had a safe and prosperous year. As thrilling as 2020 was, we were delighted to see the people's response for [PyCon India 2020](https://in.pycon.org/2020)```

> We are very excited to announce that [**PyCon India**](https://inpycon21.netlify.app/) is back online for 2021.

Reviewer: Don't think we should redirect to this link. 
This is temporary. Blog will break later on.

Author: I was thinking we would update this when we get the final website and shift to it. 

But I can remove the link if you want.

Reviewer: Then how about we just point it to https://in.pycon.org/

Once it's deployed today, we can publish this. Otherwise it's more work for you to send another patch later. Upto you.

Author: yeah, sure. Sounds like a good idea.

Get ready for 4 days of non-stop fun and learning with 4 conference tracks, sessions, BOFs, community booths, workshops, and devsprints.

> Accessing the situation with COVID-19 and the rate of vaccination, we have decided to keep PyCon India 2021 as an online event to ensure the safety of everyone involved. We know you all are excited to meet everyone in person and celebrate Python in the "real world", but everyone's health and safety are the utmost priority at this time. 

Reviewer 2: ```suggestion
Assessing the situation with COVID-19 and the rate of vaccination, we have decided to keep PyCon India 2021 as an online event to ensure the safety of everyone involved. We know folks were excited to meet everyone in person and celebrate Python in the "real world" by having an offline PyCon India 2021. We do too but maintaining everyone's health and safety is our utmost priority at all times. 
```

Author: "We were too,"?

And how about removing "at all times". Just keep ...health is our utmost priority?

## Dates

> The conference will be held from September 16 to 19, 2021.

Reviewer 2: ```suggestion
The conference will be held from September 16th to 19th, 2021.
```

* Conference Days: September 16 and 17
* Workshops: September 18
* Devsprints: September 18 and 19

**Ticket sales will be going live on April 15, 2021.**

> ** The format for devsprints is still being planned and can be changed anytime before the conference. 

Reviewer 2: `**` Was these intended?

Author: Not really. It was supposed to be bold.

## Platform

> With our amazing experience last year, and great feedback from the attendees, we have decided to use [Hopin](https://hopin.com/) as the platform for the conference again. 

Reviewer 2: ```suggestion
From the feedback received from the attendees, we have decided to use [Hopin](https://hopin.com/) as the platform to organize the conference again. 
```

Reviewer 2: No mention of Zulip?

Author: How about this?

```suggestion
From the feedback received from the attendees, we have decided to use [Hopin](https://hopin.com/) as the platform to organize the conference again.

Along with Hopin, our communication platform before, and during the conference will be [Zulip](https://zulip.com/). Feel free to join the [PyCon India Zulip channel](https://pyconindia.zulipchat.com/) anytime, if you want updates regarding the conference. 
```

## Call for Proposals

We will be starting our call for proposals soon.  In case you are excited about teaching or talking about Python, no matter whether you started recently or have been using Python for a decade, make sure you put in a talk proposal. No one became an expert in a day - neither at presenting nor at programming. So don't be shy and don't think too much, just put in a proposal.

> * CFP Opens: March 01
> * CFP Closes: May 15
> 

Reviewer 2: ```suggestion
* CFP Opens: 1st March 2021
* CFP Closes: 15th May 2021

```

Do we really want these here after what happened last year? Changing or extending dates gets people confused and then reach out that they thought X was the data for Y

Author: It was public on the inpycon2021-tasks readme, so I added it. We can remove it if you want and release the dates with the CFP announcement.

Author: ```suggestion
* CFP Opens: First week of March 2021
* CFP Closes: 15th May 2021

```

## Sponsorship and Community Partners

> From our experience organizing the conference online last year, we have decided to not take any sponsorships for this year.

Reviewer 2: ```suggestion
This year, the team has collectively decided to not take any sponsorships this year for running the logistics and organizing of PyCon India 2021.
```

Author: ```suggestion
This year, the team has collectively decided to not take any sponsorships for running the logistics and organizing of PyCon India 2021.
```

> On the other hand, we are looking forward to the support from the various Python communities across India in helping us organize, and manage the conference. 

Reviewer 2: ```suggestion
We are looking forward to the support from the various Python communities across India joining hands to help us prepare, organize, and manage the mega-conference this year! 
```

Author: ```suggestion
We are looking forward to the support from the various Python communities across India joining hands to help us prepare, organize, and manage the mega-conference this year! 

Looking forward to meeting you all during the conference. 

**PyCon India 2021 Team**
```

Title: Rejoice! PyCon India is back for 2021
Date: 2021-02-25 17:00
Author: Nikhil Maan (@Sc0rpi0n101)
Category: 2021

> Slug: announce-pycon-india-2021

Reviewer 3: ```suggestion
Slug: pycon-india-announcement
```

Summary: Announcing the return of PyCon India as an online conference for 2021.

> From all of us at the PyCon India team, we wish you've had a safe and prosperous year. As thrilling as 2020 was, we were delighted to see people's response to [PyCon India 2020](https://in.pycon.org/2020)

Reviewer 3: ```suggestion
From all of us at the PyCon India team, we wish you've had a safe and prosperous year. As thrilling as 2020 was, we were delighted to see people's response to [PyCon India 2020](https://in.pycon.org/2020).
```
